### PR TITLE
Jianyang wfsim tests

### PR DESCRIPTION
**Pull request by KaraMelih**
**What is the problem / what does the code in this PR do**
Single Electron shape data-simulation matching improvements
[Jianyang's note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:jianyang:wfsim_se_comparison)
[Melih's note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:kara_se_shape_wfsim_matching)
**WFSim needs you:**
 - Please add a test for this PR, as a bare minimum, make sure it's covered in coveralls!
 - Can you add a docsting to all your functions?

*Pay attention:*
 - Due to databases being needed for testing, making a PR from your own fork will typically NOT run the tests. If you then merge master might break

### CODE REVIEW

**Review by JYangQi00 — APPROVED** [review]

**Comment by JYangQi00** [comment]
Hi Tianyu, the note here:
https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:jianyang:wfsim_se_comparison
already assumes a liquid level of 5.1mm

File: files/XENONnT_wfsim_config.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
 {
     // Configuration flags
     "s1_model_type": "simple",
+    "s2_time_model": "s2_time_spread around zero",
```
Comment (KaraMelih) [inline files/XENONnT_wfsim_config.json:11]: Isn't it better if the default is `'optical_propagation'` ?
We also set that for the sr0 config [here](https://github.com/XENONnT/private_nt_aux_files/blob/02edbf6ba2e977aabc5e412d8558864ddd9497d4/sim_files/fax_config_nt_sr0_v0.json#L25)

Comment (zhut19) [reply files/XENONnT_wfsim_config.json:11]: Hi, This part of [the test](https://github.com/XENONnT/WFSim/blob/19dde4b1a7b833cf41c823c0a4fecfad5f386b14/tests/test_wfsim.py#L62) won't be loading any data from our database, so we can't use optical propagation. While the [test_sim_nT_advanced](https://github.com/XENONnT/WFSim/blob/19dde4b1a7b833cf41c823c0a4fecfad5f386b14/tests/test_wfsim.py#L104) will use sr0 config and thus also test the optical map. And this way we can also maximize test coverage.

**Comment by zhut19** [comment]
@KaraMelih I also uploaded the optical map in a format that can be directly used by straxen in https://github.com/XENONnT/private_nt_aux_files/pull/167. Can you help double-check it's consistent with your result? Thanks! Also, that pr has to be merged before the tests here pass since the test on that part fetches the file through our MongoDB, which only updates with the master branch.
